corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1646,7 +1646,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         },
         getPersistantMenuRegionWidth: function () {
             let persistantMenuRegionWidth = sessionStorage.getItem('persistantMenuRegionWidth');
-            if (!persistantMenuRegionWidth) {
+            if (!persistantMenuRegionWidth || persistantMenuRegionWidth === '0') {
```

Comment:
Was this change to fix the issue I screenshot above? When would it be 0? i'm wondering if it could potentially be any other incorrect value